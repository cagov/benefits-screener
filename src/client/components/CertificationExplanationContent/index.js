import { Trans, useTranslation } from "react-i18next";
import React from "react";

function CertificationExplanationContent() {
  const { t } = useTranslation();

  return (
    <div className="gray-box">
      <h4>
        <Trans t={t} i18nKey="certification-header1" />
      </h4>
      <p>
        <Trans t={t} i18nKey="certification-p1" />
      </p>
      <ul>
        <li>
          <Trans t={t} i18nKey="certification-list1-item1" />
        </li>
        <li>
          <Trans t={t} i18nKey="certification-list1-item2" />
        </li>
        <li>
          <Trans t={t} i18nKey="certification-list1-item3" />
        </li>
        <li>
          <Trans t={t} i18nKey="certification-list1-item4" />
        </li>
        <li>
          <Trans t={t} i18nKey="certification-list1-item5" />
        </li>
      </ul>
      <p>
        <Trans t={t} i18nKey="certification-p2">
          The fastest way to certify is on{" "}
          <a href="https://portal.edd.ca.gov/WebApp/Login?resource_url=https%3A%2F%2Fportal.edd.ca.gov%2FWebApp%2FHome">
            UI Online
          </a>
          . You can also do this by phone by calling 1-866-333-4606, or by
          mailing the paper form.
        </Trans>
      </p>
    </div>
  );
}

export default CertificationExplanationContent;
