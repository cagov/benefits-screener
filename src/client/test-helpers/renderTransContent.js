import React from "react";
import { mount } from "enzyme";
import { I18nextProvider } from "react-i18next";
import i18n from "./i18nForTests";

function renderTransContent(Component, props = {}) {
  const wrapper = mount(
    <I18nextProvider i18n={i18n}>
      <Component {...props} />
    </I18nextProvider>
  );

  // Filter out the very verbose I18nextProvider component wrapper
  return wrapper.find(Component);
}

module.exports = renderTransContent;
