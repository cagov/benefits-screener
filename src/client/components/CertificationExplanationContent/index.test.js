import renderTransContent from "../../test-helpers/renderTransContent";
import CertificationExplanationContent from "./index";

describe("<CertificationExplanationContent />", () => {
  it("renders the component", async () => {
    const content = renderTransContent(CertificationExplanationContent);
    expect(content).toMatchSnapshot();
  });
});
