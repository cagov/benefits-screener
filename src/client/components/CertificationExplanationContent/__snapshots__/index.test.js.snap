// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CertificationExplanationContent /> renders the component 1`] = `
<CertificationExplanationContent>
  <div
    className="gray-box"
  >
    <h4>
      <Trans
        i18nKey="certification-header1"
        t={[Function]}
      >
        What is certification?
      </Trans>
    </h4>
    <p>
      <Trans
        i18nKey="certification-p1"
        t={[Function]}
      >
        Certification is the required process of updating the EDD every two weeks with your unemployment status with basic eligibility information:
      </Trans>
    </p>
    <ul>
      <li>
        <Trans
          i18nKey="certification-list1-item1"
          t={[Function]}
        >
          Are you physically able to work?
        </Trans>
      </li>
      <li>
        <Trans
          i18nKey="certification-list1-item2"
          t={[Function]}
        >
          Are you available for work?
        </Trans>
      </li>
      <li>
        <Trans
          i18nKey="certification-list1-item3"
          t={[Function]}
        >
          Are you ready and willing to accept work immediately?
        </Trans>
      </li>
      <li>
        <Trans
          i18nKey="certification-list1-item4"
          t={[Function]}
        >
          Did you refuse work
        </Trans>
      </li>
      <li>
        <Trans
          i18nKey="certification-list1-item5"
          t={[Function]}
        >
          Did you work and earn wages? If so, how much?
        </Trans>
      </li>
    </ul>
    <p>
      <Trans
        i18nKey="certification-p2"
        t={[Function]}
      >
        The fastest way to certify is on 
        <a
          href="https://portal.edd.ca.gov/WebApp/Login?resource_url=https%3A%2F%2Fportal.edd.ca.gov%2FWebApp%2FHome"
          key="1"
        >
          UI Online
        </a>
        . You can also do this by phone by calling 1-866-333-4606, or by mailing the paper form.
      </Trans>
    </p>
  </div>
</CertificationExplanationContent>
`;
